designs/Verify-Verbosity.md

```diff
@@ -0,0 +1,302 @@
+  Valid from: 12/22/2017 4:00:00 PM to 3/22/2029 4:59:59 PM
+
+Signature type: Repository
+nuget-v3-service-index-url: https://api.nuget.org/v3/index.json
+nuget-package-owners: Microsoft, nuget
+Verifying the repository countersignature with certificate: 
+  Subject Name: CN=NuGet.org Repository by Microsoft, O=NuGet.org Repository by Microsoft, L=Redmond, S=Washington, C=US
+  SHA1 hash: 8FB6D7FCF7AD49EB774446EFE778B33365BB7BFB
+  SHA256 hash: 0E5F38F57DC1BCC806D8494F4F90FBCEDD988B46760709CBEEC6F4219AA6157D
+  Issued by: CN=DigiCert SHA2 Assured ID Code Signing CA, OU=www.digicert.com, O=DigiCert Inc, C=US
+  Valid from: 4/9/2018 5:00:00 PM to 4/14/2021 5:00:00 AM
+Timestamp: 12/9/2020 2:32:12 PM
+Verifying repository countersignature's timestamp with timestamping service certificate: 
+  Subject Name: CN=Symantec SHA256 TimeStamping Signer - G3, OU=Symantec Trust Network, O=Symantec Corporation, C=US
+  SHA1 hash: A9A4121063D71D48E8529A4681DE803E3E7954B0
+  SHA256 hash: C474CE76007D02394E0DA5E4DE7C14C680F9E282013CFEF653EF5DB71FDF61F8
+  Issued by: CN=Symantec SHA256 TimeStamping CA, OU=Symantec Trust Network, O=Symantec Corporation, C=US
+  Valid from: 12/22/2017 4:00:00 PM to 3/22/2029 4:59:59 PM
+
+Successfully verified package 'NuGet.Common.5.9.0-preview.2'.
+```
+
+</details>
+
+#### `--verbosity detailed`
+
+<details>
+<summary>output</summary>
+
+```
+Verifying NuGet.Common.5.9.0-preview.2
+<PATH>\nuget.common.5.9.0-preview.2.nupkg
+Signature Hash Algorithm: SHA256
+
+Signature type: Author
+Verifying the author primary signature with certificate: 
+  Subject Name: CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US
+  SHA1 hash: F404000FB11E61F446529981C7059A76C061631E
+  SHA256 hash: 3F9001EA83C560D712C24CF213C3D312CB3BFF51EE89435D3430BD06B5D0EECE
+  Issued by: CN=DigiCert SHA2 Assured ID Code Signing CA, OU=www.digicert.com, O=DigiCert Inc, C=US
+  Valid from: 2/25/2018 4:00:00 PM to 1/27/2021 4:00:00 AM
+trace:       Subject Name: CN=DigiCert SHA2 Assured ID Code Signing CA, OU=www.digicert.com, O=DigiCert Inc, C=US
+trace:       SHA1 hash: 92C1588E85AF2201CE7915E8538B492F605B80C6
+trace:       SHA256 hash: 51044706BD237B91B89B781337E6D62656C69F0FCFFBE8E43741367948127862
+trace:       Issued by: CN=DigiCert Assured ID Root CA, OU=www.digicert.com, O=DigiCert Inc, C=US
+trace:       Valid from: 10/22/2013 5:00:00 AM to 10/22/2028 5:00:00 AM
+trace: 
+trace:             Subject Name: CN=DigiCert Assured ID Root CA, OU=www.digicert.com, O=DigiCert Inc, C=US
+trace:             SHA1 hash: 0563B8630D62D75ABBC8AB1E4BDFB5A899B24D43
+trace:             SHA256 hash: 3E9099B5015E8F486C00BCEA9D111EE721FABA355A89BCF1DF69561E3DC6325C
+trace:             Issued by: CN=DigiCert Assured ID Root CA, OU=www.digicert.com, O=DigiCert Inc, C=US
+trace:             Valid from: 11/9/2006 4:00:00 PM to 11/9/2031 4:00:00 PM
+trace: 
```

Comment:
Is the "trace:" prefix useful to customers?

Reply:
It looks the prefix "trace: " is configured [here](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/CommandOutputLogger.cs#L79). This was defined in mid of 2015 in this [commit](https://github.com/NuGet/NuGet.Client/commit/ee9ed7fea1f92d2131502af7e5062c1b080f1c2b). I would consider this as a out-of-scope for this spec because any change affects all the `dotnet` commands. Please let me know your thoughts. I will be happy to create a new issue if we would like to change this functionality.